config/sync/uhsg_role_auto_assign.settings.yml

```diff
@@ -6,8 +6,8 @@ group_to_roles:
     group_name: hy-ypa-opa-kosu
     rid: content_editor
   -
-    group_name: grp-guide-editor
+    group_name: grp-guide-editor-dev
```

Comment:
The group changes came from database (development), i reverted them since i'm not sure they are suppose to be changed. Will double-check from production.